.github/workflows/test_and_release.yml

```diff
@@ -83,6 +83,7 @@ jobs:
           dpns-top-level-identity-id: ${{ env.dpns-top-level-identity-id }}
           dpns-top-level-identity-private-key: ${{ env.dpns-top-level-identity-private-key }}
           platform-test-suite-branch: ${{ steps.platform-branches.outputs.testsuite-branch }}
+          scope: platform
```

Comment:
I revisited tests and now I'm not sure that we need the scope for SDK anymore.

Reply:
OK, should I close these PRs and remove it from drive CI?

Reply:
Just close this one.